plugins/README.md

```diff
@@ -37,14 +39,37 @@ plugins.
 - [``@blockly/theme-modern``](theme-modern/): A Blockly modern theme.
 - [``@blockly/theme-dark``](theme-dark/): A Blockly dark theme.
 
-### Other
-
+### Block Extension
+
+- [``@blockly/block-extension-tooltip``](block-extension-tooltip/): A Blockly
+block extension that adds support for custom tooltip rendering.
+
+### Workspace
+
+- [``@blockly/continuous-toolbox``](continuous-toolbox/): A Blockly plugin
+that adds a continous-scrolling style toolbox and flyout
+- [``@blockly/disable-top-blocks``](disable-top-blocks/):  A Blockly plugin that
+shows the 'disable' context menu option only on non-orphan blocks.
+- [``@blockly/fixed-edges``](fixed-edges/):  A Blockly MetricsManager for
+configuring fixed sides.
+- [``@blockly/keyboard-navigation``](keyboard-navigation/): A Blockly plugin
+that adds keyboard navigation support.
+- [``plugin-nominal-connection-checker``](nominal-connection-checker/): A
```

Comment:
This plugin is private, I wasn't sure whether we wanted to exclude it from the README for that reason

Reply:
We don't have to exclude it, but put "(private; in development)" at the end of the line.